### DISCUSSION TITLE
Feature: alignment extension

### DISCUSSION THREAD

**Discussion by zhaoyao91** [remirror/remirror#227] [Ideas]
## Description

Alignment seems a basic function in many text editors.

As image is inline in remirror, supporting alignment for blocks would make the image positioning much easier.

## Possible Implementation

Currently I have no idea how to implement it. Marks are for inline styling, but alignment is block level styling. It would be tedious to extend every block types to support it.

## Checklist

- [ ] I have read the [**contributing**](https://github.com/remirror/remirror/blob/canary/docs/contributing.md) document.

**Comment by ifiokjr** [comment]
Sounds good. Might be an idea for a new formatting group of extensions which cover all these scenarios.

**Reply by anotheri** [reply]
I would love to have it! 🙏
